### DISCUSSION TITLE
My app is dying!!!Calculating area from DXF Files

### DISCUSSION THREAD

**Discussion by AdilaK** [mozman/ezdxf#1208] [General]
I made a DXF Quoting app for our machine shop using Python and ezdxf

However, the area calculated from the DXF is wrong, resulting in wrong pricing for quoted parts. I m now trying to see if the length(perimeter) of the drawing is correct, and then go from there, any other better way to do this??

**Comment by mozman** [comment]
The only area calculation that `ezdxf` provides is the area of ​​a polygon, and this is a standard algorithm, see section `Area`: https://en.wikipedia.org/wiki/Polygon

**Comment by AdilaK** [comment]
Thank you for the reply. Any other way to do it, if ezdxf can't do it, pretty much my idea is dead it seems :(
